openapi/openapi.yml

```diff
@@ -1370,15 +1327,19 @@ components:
       schema:
         type: integer
         default: 10
-    postgres_database_id:
-      description: Postgres database ID
+    postgres_database_identifier:
```

Comment:
Up to you, but maybe `postgres_database_reference` instead?

Reply:
I like that better. I didn't love identifier because it felt to close to id and seemed like it could create confusion, but wasn't coming up with a suitable alternative in the moment. I changed this to reference in https://github.com/ubicloud/ubicloud/pull/2907/commits/15994d3999d9c1d0eaab7eab2afa0dce4764e5ea